### ISSUE TITLE
Distribute maude in this repo

### ISSUE THREAD

**Issue by katrielalex** [tamarin-prover/bin-dists#2]
Every time I use Tamarin on a new computer I get bitten by the requirement for maude. I think a minimal install at the moment is

```
hub clone tamarin-prover/bin-dists
curl http://maude.cs.uiuc.edu/download/current/maude.tar.gz | tar xz
ln -s maude/maude.$OS $SOMEWHERE
ln -s maude/prelude.maude $SOMEWHERE
```
If maude were included in this repo (and, probably, a small amount of logic added to tamarin), it should be possible to skip the last three lines. It looks like maude is GPL-ed so I think it's legally fine.

Does this sound like a sensible suggestion?

**Comment by meiersi** [comment]
Yes, I think that makes a lot of sense. Less hassle and a better
experience. What about GraphViz then?

Katriel Cohn-Gordon <notifications@github.com> schrieb am Fr., 20. Mai 2016
11:29 AM:

**Comment by katrielalex** [comment]
I seem to have dot already installed, but I don't see why it shouldn't be here too. Probably worth making folders `tools/maude/maude.{linux,...}` and `tools/graphviz/dot`.

(I heard the graphviz dependency is on its way out though... :smile:)

**Comment by meiersi** [comment]
Ok. Cool @ GraphViz replacement :)

Katriel Cohn-Gordon <notifications@github.com> schrieb am Fr., 20. Mai 2016
1:53 PM:
